### ISSUE TITLE
Allow exclusion from results only

### ISSUE THREAD

**Issue by manuelpichler** [phpmd/phpmd#41]
## Short description

It would be nice if excluded classes were parsed and analyzed, but no problems were reported for them. It's mainly related to frameworks or software libraries - I'd like to check for eg. DepthOfInheritance violations but at the same time I'm not interested in errors in framework design. 
## Steps to reproduce

``` php
File: classes.php
<?php
class A {}
class B extends A {}
class C extends B {}
class D extends C {}
class E extends D {}
class F extends E {}
```

File: testcase.php

``` php
<?php
require_once __DIR__ . 'classes.php';
class G extends F {}
```

phpmd --suffixes php --exclude classes . text rulesets/design.xml
## Expected behavior

testcase.php:3  The class G has 6 parents. Consider to reduce the depth of this class hierarchy.
## Actual behavior

No problem found

[Synced from PivotalTracker: 'http://www.pivotaltracker.com/story/show/9355849']
